document-portal/document-portal.c

```diff
@@ -1297,6 +1300,15 @@ on_bus_acquired (GDBusConnection *connection,
       g_warning ("error: %s", error->message);
       g_error_free (error);
     }
+
```

Comment:
Don't we need a:
```
  g_dbus_interface_skeleton_set_flags (file_transfer,
                                       G_DBUS_INTERFACE_SKELETON_FLAGS_HANDLE_METHOD_INVOCATIONS_IN_THREAD);
```
Here? Otherwise all the file transfer calls will happen on the main thread, and the locks they take can block it.